src/pe_mailer/email_reports.py

```diff
@@ -483,9 +483,11 @@ def main():
     # Assign validated arguments to variables
     log_level: str = validated_args["--log-level"]
 
-    # Set up logging
+    # Setup logging to central file
     logging.basicConfig(
-        format="%(asctime)-15s %(levelname)s %(message)s", level=log_level.upper()
+        filename="pe_reports_logging.log",
```

Comment:
How come you are not using the `CENTRAL_LOGGING_FILE` variable here?  Also why are you using `filemode="a"` and different formatting options in `src/pe_reports/__init__.py`, but not here?

Reply:
@dav3r 
I have updated the logging filename and formatting to match throughout the project per comit [7ccf5c9](https://github.com/cisagov/pe-reports/pull/169/commits/7ccf5c9c137ff89f5102120ec00d4ba80ff2c71d) I use the filemode='a' to append the file and as I understand this is what the mode that we want so that we have a continually growing log file. Do you have a recommendation as to another filemode that we should be using?

Reply:
@dav3r 
I did leave the individual logging.basicConfig in place thinking that at some point we may want to completely separate the logging by module.

Reply:
That's fine, though my personal preference would be to fully centralize the logging now to DRY out the code and then change it later if you need to.